include/crab/domains/union_find_domain.hpp

```diff
@@ -514,6 +514,21 @@ template <class Variable, class Domain> class union_find_domain {
     m_parents.insert({y, rep_x});
   }
 
+  std::vector<variable_t> get_equiv_vars_by_var(const variable_t &x) {
```

Comment:
I add this method because I need to get all region variables used by a single object.